pom.xml

```diff
@@ -91,6 +91,8 @@
                 <configuration>
                     <excludes>
                         <exclude>**/Main*</exclude>
+                        <exclude>**/FruitTransaction*</exclude>
+                        <exclude>**/OperationStrategyImpl*</exclude>
```

Comment:
why do we need to exclude OperationStrategyImpl? I think we could add tests to this class